server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoErrorRetryAnalyzer.ts

```diff
@@ -30,9 +30,20 @@ export class MongoErrorRetryAnalyzer {
 
     public shouldRetry(error: Error): boolean {
         const rule = this.getRetryRule(error);
+        const sanitizedError = {
```

Comment:
Is this to avoid logging potential customer content in the error object? If yes, can we re-use the same error sanitizer that Arik is working on?

Reply:
@shubhi1092, I think this is still a good idea to log only what is necessary irrespective of whether the sanitizer earlier had already sanitized the error or not.

Reply:
I wasnt aware if the error is already sanitized before shouldRetry, but thanks for clarifying and that makes sense.